app/controllers/exchanges/employer_applications_controller.rb

```diff
@@ -142,12 +143,65 @@ def revise_end_date
     redirect_to exchanges_hbx_profiles_root_path, flash[:error] => "#{application.benefit_sponsorship.legal_name} - #{l10n('exchange.employer_applications.unable_to_change_end_date')}"
   end
 
+  def download_v2_xml
```

Comment:
Both this and the upload action should be placed behind an Operation with a Contract and proper parameter validation and specs.